catalyst-gateway/bin/src/event_db/legacy/queries/event/config.rs

```diff
@@ -0,0 +1,81 @@
+//! Config Queries
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+use tracing::error;
+
+use crate::event_db::{Error, EventDB};
+
+#[async_trait]
+#[allow(clippy::module_name_repetitions)]
+/// Config Queries Trait
+pub(crate) trait ConfigQueries: Sync + Send + 'static {
```

Comment:
I know mainly it is already done for all legacy queries, but I think we dont actually need to specify traits for queries.
Initial idea which was come from `cat-data-service` to provide these traits is that it would be a possibility to write a mocks for testing purposes. As seems we are not going to make tests that way, these traits are redundant anymore, so my suggestion just get rid of them.

Reply:
I will refactor trait stuff in two PRs.